### ISSUE TITLE
Test Issue

### ISSUE THREAD

**Issue by gitScald** [CONCORDIA-2/Unnamed#1]
This is a dummy issue to show how they work on GitHub.

Ideally, as soon as an issue is created, it should be labeled, affected a milestone, and linked to the project board (all on the side menu to the right). Issues can be assigned to people; try to leave that to your "role" representatives.

Here's a brief breakdown of the labels created so far:

Category | Label | Comments
---------- | ------ | ------------
**Domain** | **art** | An issue pertaining to art (models, textures, concept art...).
**Domain** | **code** | An issue pertaining to code (scripts, editor behavior, shaders...).
**Domain** | **design** | An issue pertaining to flaws in the project's design (mechanics revision, character control issues...).
**Feedback** | **needs info** | An issue that is missing information for it to be resolved. Consider adding screenshots, or means to reproduce it.
**Feedback** | **no fix** | An issue that has been determined to be skipped, either until the next weekly sprint, or scrapped altogether in hopes of reducing scope.
**Feedback** | **on hold** | An issue that is blocked by external forces (software license purchase, supervisor approval...).
**Status** | **bug** | A bug in the code, or in the asset, that causes unexpected behavior.
**Status** | **critical** | A more serious version of a bug, with a high priority to fix.
**Status** | **missing** | A missing feature that was missed, forgotten, or delayed in a previous weekly sprint.
**Status** | **needs test** | An issue that has been worked on, and whose fix requires to be validated.
**Suggestion** | **addition** | A feature that would be worth including, or worth discussing at a meeting.
**Suggestion** | **optimization** | Any optimization of the source code (refactoring, efficiency...) or of an asset (reduction of polygon counts, file compression...).

Issues (like a lot of things on GitHub) use [Markdown](https://guides.github.com/features/mastering-markdown/) for formatting. Screenshots are helpful to help debug code.

**Comment by gitScald** [comment]
This is a comment. Comments are incredibly useful to create a discussion around the issue.

Mentions (_@user_) can be used effectively here!
